opds2_import.py

```diff
@@ -845,6 +876,28 @@ def extract_feed_data(self, feed, feed_url=None):
                 publication_metadata.primary_identifier.identifier
             ] = publication_metadata
 
+        node_finder = NodeFinder()
+
+        for error in parser_result.errors:
+            publication = node_finder.find_parent_or_self(
+                parser_result.root, error.node, opds2_ast.OPDS2Publication
+            )
+
+            if publication:
+                identifier = self._extract_identifier(publication)
+
+                if identifier:
```

Comment:
What happens to errors for which there is no `publication` or no `identifier`? It looks like it would be quietly ignored.